apps/designer/app/designer/features/style-panel/controls/font-weight/is-supported-font-weight.ts

```diff
@@ -0,0 +1,30 @@
+import type { Asset } from "@webstudio-is/asset-uploader";
+
+export const isSupportedFontWeight = (
+  asset: Asset,
+  weight: string,
+  currentFamily: string
+) => {
+  const isCurrentFamily =
+    "meta" in asset &&
+    "family" in asset.meta &&
+    asset.meta.family === currentFamily;
+
+  if (isCurrentFamily === false) {
+    return false;
+  }
+
+  const weightNumber = Number(weight);
+  const { meta } = asset;
+
+  if ("variationAxes" in meta) {
+    const { variationAxes } = meta;
+    return (
+      variationAxes.wght !== undefined &&
+      variationAxes.wght.min <= weightNumber &&
+      variationAxes.wght.max >= weightNumber
+    );
+  }
+
+  return "weight" in meta && meta.weight === weightNumber;
```

Comment:
`return meta.weight === weightNumber`

Reply:
same problem, ts doesn't allow

Reply:
feel free to commit if you want to figure it out

Reply:
the problem there is that meta can be variable font meta object that has no weight and ts doesn't know which one it is for some reason, even though the check above should have taught it at this point

Reply:
Oh I figured it out, it was confused because asset can also be image asset